package.json

```diff
@@ -15,14 +15,16 @@
     "@babel/preset-env": "^7.18.9",
     "dotenv": "^16.0.1",
     "nodemon": "^2.0.19",
-    "twilio": "^3.82.2"
+    "twilio": "^3.83.3"
   },
   "dependencies": {
     "@google-cloud/speech": "^4.10.2",
     "actions-on-google": "^3.0.0",
     "cookie-parser": "^1.4.6",
     "express": "^4.18.1",
+    "fs": "0.0.1-security",
     "superagent": "^8.0.0",
+    "twilio": "^3.83.3",
```

Comment:
I don't think we need this in dependences _and_ dev dependences

Reply:
True that.